server/protocols/jmap/src/main/java/org/apache/james/jmap/utils/DependencyGraph.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
content is the same. Only copyright notice formatting had been lost from my IDE project settings.
Fixed.